bin/check_tron_jobs.py

```diff
@@ -176,32 +181,81 @@ def is_job_scheduled(job_runs):
     return None
 
 
+def _get_delta_seconds_from_str(expected_runtime_str):
+    expected_runtime = config_utils.valid_time_delta(
+        expected_runtime_str, config_utils.NullConfigContext
+    )
+    return timeutils.delta_total_seconds(expected_runtime)
+
+
 def is_job_stuck(job_runs):
     next_run_time = None
+
+    expected_runtime_str = job_runs.get('expected_runtime', None)
+    if expected_runtime_str is not None:
+        seconds = _get_delta_seconds_from_str(expected_runtime_str)
+    actions_expected_runtime = job_runs.get('actions_expected_runtime', {})
+
     for run in sorted(
         job_runs['runs'],
         key=lambda k: k['run_time'],
         reverse=True,
     ):
         if run.get('state', 'unknown') == "running":
+            # check if runtime exceeds expected_runtime
+            if expected_runtime_str is not None:
+                starttime = run.get('start_time', None)
```

Comment:
you made a `is_action_exceeding_expected_runtime`, can you wrap this up into an `is_job_exceeding_expected_runtime` ? I think it would make it easier to read

Reply:
job_run maybe